libr/util/p_format.c

```diff
@@ -1015,8 +1015,15 @@ int r_print_format_struct_size(const char *f, RPrint *p, int mode) {
 		mode &= ~R_PRINT_UNIONMODE;
 	}
 
+	i = 0;
+	if (fmt[i] >= '0' && fmt[i] <= '9') {
+		while (fmt[i] >= '0' && fmt[i] <= '9') {
+			i++;
+		}
+	}
+
 	r_str_word_set0 (args);
-	for (i=0; i < strlen (fmt); i++) {
+	for (; i < strlen (fmt); i++) {
```

Comment:
dont use strlen in for loops, otherwise it will be called on every iteration when you only need to compute this length before entering the loop